iphone/Maps/UI/DocumentPicker/DocumentPicker.swift

```diff
@@ -0,0 +1,28 @@
+typealias URLsCompletionHandler = ([URL]) -> Void
+
+final class DocumentPicker: NSObject {
+  static let shared = DocumentPicker()
+  private var completionHandler: URLsCompletionHandler?
+
+  func present(from rootViewController: UIViewController,
+               fileTypes: [FileType] = [.kml, .kmz, .gpx],
+               completionHandler: @escaping URLsCompletionHandler) {
+    self.completionHandler = completionHandler
+    let documentPickerViewController: UIDocumentPickerViewController
+    if #available(iOS 14.0, *) {
+      documentPickerViewController = UIDocumentPickerViewController(forOpeningContentTypes: fileTypes.map(\.utType), asCopy: true)
```

Comment:
> Don’t copy the document if you can avoid it.

Is copying really necessary? What does it do? Where copied files are stored and how they're deleted?

Reply:
It is not necessary, but I think that it's safer to pass not the document itself but the copy. We load multiple files at once and maybe it's not the best case to store the URLs to the source files during the importing process. 
For example, we can import the file from the cloud directory but it can be deleted during the importing process (extremely rare case).

This files will be placed in the tmp directory and we can delete them by our own or the system will delete them.

Reply:
Copying, especially for larger files, takes additional time. Does it make sense to start without copying, and only add it later if it is really necessary?